package.json

```diff
@@ -304,7 +304,10 @@
 		"@wordpress/base-styles": "4.13.0",
 		"@wordpress/interface@^4.8.0": "patch:@wordpress/interface@npm%3A4.8.0#./.yarn/patches/@wordpress-interface-npm-4.8.0-8a39ad37cf.patch",
 		"@wordpress/data-controls": "2.22.0",
-		"@wordpress/element": "4.20.0"
+		"@wordpress/element": "4.20.0",
+		"@wordpress/block-editor": "10.4.0",
+		"@wordpress/block-library": "7.17.0",
```

Comment:
I wonder if we could pin to the versions we use broadly right now:

* `@wordpress/bock-editor@10.5.0`
* `@wordpress/block-library@7.19.0`

The other forced resolutions here seem to match what we're already using at this time.

This is, of course, temporary, since as @noahtallen mentioned in p4TIVU-auj-p2 we are in the middle of upgrading Calypso to the latest `@wordpress` packages and upgrading to React 18 in the meantime.

Reply:
Unfortunately, changing to those versions starts to trigger console errors--there are probably other package version changes that would need to be in a new version of isolated-block-editor to align with those versions of block-editor and block-library.

Using `wordpress/block-editor@10.5.0` and `@wordpress/block-library@7.19.0`, here's the error I see, when using slash to insert a block: 2ff1d-pb. I also tried changing just `wordpress/block-editor@10.5.0` (with `@wordpress/block-library@ 7.17.0`), but that results in a build error.